exporter/awsemfexporter/datapoint.go

```diff
@@ -81,15 +81,11 @@ type numberDataPointSlice struct {
 
 // histogramDataPointSlice is a wrapper for pmetric.HistogramDataPointSlice
 type histogramDataPointSlice struct {
-	// Todo:(khanhntd) Calculate delta value for count and sum value with histogram
```

Comment:
I removed these comments because I don't think it is accurate to say that it is a "bug" that we need to change the behavior on.  Changing the behavior here is a breaking change for customers.  To implement the "issue" we would need to make this configurable or onboard highly specific use-cases (like the one in this PR)